core/commands/wallet.go

```diff
@@ -307,13 +324,93 @@ var walletTransactionsCmd = &cmds.Command{
 		}
 		return cmds.EmitOnce(res, txs)
 	},
-	Type: []*walletpb.Transaction{},
+	Type: []*walletpb.TransactionV1{},
+}
+
+var walletTransferCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline:          "Send to another BTT wallet",
+		ShortDescription: "Send to another BTT wallet from current BTT wallet",
+	},
+	Arguments: []cmds.Argument{
+		cmds.StringArg("to", true, false, "address of another BTFS wallet to transfer to."),
+		cmds.StringArg("amount", true, false, "amount of µBTT (=0.000001BTT) to transfer."),
+	},
+	Options: []cmds.Option{},
+	Run: func(req *cmds.Request, res cmds.ResponseEmitter, env cmds.Environment) error {
+		n, err := cmdenv.GetNode(env)
+		if err != nil {
+			return err
+		}
+		cfg, err := n.Repo.Config()
+		if err != nil {
+			return err
+		}
+		amount, err := strconv.ParseInt(req.Arguments[1], 10, 64)
+		if err != nil {
+			return err
+		}
+		ret, err := wallet.TransferBTT(req.Context, n, cfg, nil, "", req.Arguments[0], amount)
+		if err != nil {
+			return err
+		}
+		return cmds.EmitOnce(res, &TransferResult{
+			Result:  ret.Result,
+			Message: ret.Message,
+		})
+	},
+	Type: &TransferResult{},
 }
 
-type Transaction struct {
-	Datetime time.Time
-	Amount   int64
-	From     string
-	To       string
-	Status   string
+type TransferResult struct {
+	Result  bool
+	Message string
+}
+
+const privateKeyOptionName = "privateKey"
+const mnemonicOptionName = "mnemonic"
+
+var walletImportCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline:          "BTFS wallet import",
+		ShortDescription: "import BTFS wallet",
+	},
+	Arguments: []cmds.Argument{},
+	Options: []cmds.Option{
+		cmds.StringOption(privateKeyOptionName, "p", "Private Key to import."),
+		cmds.StringOption(mnemonicOptionName, "m", "Mnemonic to import."),
+	},
+	Run: func(req *cmds.Request, res cmds.ResponseEmitter, env cmds.Environment) error {
+		n, err := cmdenv.GetNode(env)
+		if err != nil {
+			return err
+		}
+
+		privKey, _ := req.Options[privateKeyOptionName].(string)
+		mnemonic, _ := req.Options[mnemonicOptionName].(string)
+		err = wallet.ImportKeys(n, privKey, mnemonic)
+		if err != nil {
+			return err
+		}
+		go func() error {
+			shutdownCmd := exec.Command("btfs", "shutdown")
```

Comment:
Why not use `restart` which handles his case?

Reply:
`restart` has side effects now. will change it.